release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2807,3 +2807,24 @@ link:https://access.redhat.com/solutions/6455311[{product-title} 4.8.17 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-8-18"]
+=== RHBA-2021:4020 - {product-title} 4.8.18 bug fix update
+
+Issued: 2021-11-02
+
+{product-title} release 4.8.18 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4020[RHBA-2021:4020] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:4019[RHBA-2021:4019] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6469021[{product-title} 4.8.18 container image list]
+
+[id="ocp-4-8-18-bug-fixes"]
+==== Bug fixes
+
+* With the deprecation of the last triggered image ID field for build configs, the image change trigger controller stopped checking the ID field prior to initiating builds. Consequently, if a build config was created and had an image change trigger start while the cluster was a level prior to {product-title} 4.8, it continuously tried to trigger builds. With this update, unnecessary attempts to trigger builds no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2006793[*BZ#2006793*])
```

Comment:
s/the last triggered image ID field / lastTriggeredImageID parameter to match what is earlier in the release notes?

s/a level prior / a version prior ?

s/unnecessary attempts / these unnecessary attempts as this fix doesn't fix *all* unnecessary attempts, just these ones.

Reply:
Thanks for those suggestions. I've made changes to address them all.

Reply:
FYI @stevsmit